internal/extract_test.go

```diff
@@ -50,7 +53,7 @@ func makeTar() (BufferReaderMaker, []byte) {
 
 	r, w := io.Pipe()
 	go func() {
-		bw := bufio.NewWriterSize(w, 4)
+		bw := bufio.NewWriterSize(w, minBufferSize)
```

Comment:
it was 4 but became 1024, is it expected?

Reply:
Yep
It's just a buffer size.
See [doc](https://pkg.go.dev/bufio#NewWriterSize)